controllers/lvmcluster_controller.go

```diff
@@ -36,19 +38,31 @@ var lvmClusterFinalizer = "lvmcluster.topolvm.io"
 
 const (
 	ControllerName = "lvmcluster-controller"
+
+	openshiftSCCPrivilegedName = "privileged"
+)
+
+type ClusterType string
+
+const (
+	ClusterTypeOCP   ClusterType = "openshift"
+	ClusterTypeOther ClusterType = "other"
```

Comment:
do we really need `other` option here? I think only `ClusterTypeOCP` should do the job.

Reply:
`other` looks a bit ambiguous as well

Reply:
It is set to a value to indicate that the check has been performed and a result has been received.